downloads.md

```diff
@@ -159,3 +159,25 @@ Installing NeoMutt is as simple as:
 ```
 xbps-install neomutt
 ```
+
+## NixOS
+
+Packagers:
+* [Charles Strahan](https://github.com/cstrahan),
+* [Christian Lask](https://github.com/hiberno),
+* [Rahul Gopinath](https://github.com/vrthra) and
+* [Erik Rybakken](https://github.com/erikryb)
+
+To install NeoMutt:
+
+add `pkgs.neomutt` into `configuration.nix` like that:
+
+```
+environment.systemPackages = [ pkgs.neomutt ];
```

Comment:
It might make sense so split the `Nix` section into two here: one for `nix-env`, the "imperative" way of using the `nixpkg` of `Neomutt` (which works on `NixOS`, but elsewhere too) and one for the more declarative way that is described and particular to `NixOS`.